components/insight/ivy.xml

```diff
@@ -61,6 +61,8 @@
     <dependency org="net.java.dev.jna" name="platform" rev="${versions.platform}"/>
     <dependency org="edu.ucar" name="grib" rev="${versions.grib}" transitive="false"/>
     <dependency org="edu.ucar" name="bufr" rev="${versions.bufr}" transitive="false"/>
+    <!-- from Java 9 on javax.activation is not included any longer -->
+    <dependency org="javax.activation" name="activation" rev="1.1.1"/>
```

Comment:
better to add ``omero.properties`` i.e. add  ``versions.activation``

Reply:
Entry for the jar is already in ``license/README``
``
  Artifacts:    activation-*.jar
  Project:      The JavaBeans(TM) Activation Framework is used by the JavaMail(TM) API to manage MIME data
  Web Site:     http://java.sun.com/javase/technologies/desktop/javabeans/jaf/index.jsp
  License:      Common Development and distribution License (CDDL) Version 1.0
  License File: cddl.txt
``